Tests/XCUITests/IntegrationTests.swift

```diff
@@ -98,9 +99,6 @@ class IntegrationTests: BaseTestCase {
         navigator.performAction(Action.OpenEmailToSignIn)
 
         mozWaitForElementToExist(app.navigationBars[AccessibilityIdentifiers.Settings.FirefoxAccount.fxaNavigationBar], timeout: TIMEOUT_LONG)
-        mozWaitForElementToExist(app.webViews.staticTexts["Continue to Firefox accounts"], timeout: TIMEOUT_LONG)
```

Comment:
is it enough by removing these lines or should we also remove the previous one?

Reply:
Yes. The browser does launch after `navigator.performAction(Action.OpenEmailToSignIn)`.